lib/src/commands/create/commands/flutter_app.dart

```diff
@@ -39,4 +36,13 @@ class CreateFlutterApp extends CreateSubCommand with OrgName {
 
     return vars;
   }
+
+  @override
+  String get defaultTemplateName => 'core';
```

Comment:
```suggestion
  String get defaultTemplateName => templates.first.name;
```

Can't this be the default either? What if we change the template name and forget it here for instance?

Reply:
>  What if we change the template name and forget it here for instance?

Then we will have broken unit tests

Reply:
Good point! But imho then we should at least use the reference like this, that way we dont have to think about it anyway :D

Reply:
Yeah, changed to be the default behavior fo the mixin